src/basics.md

```diff
@@ -278,17 +278,20 @@ fn run() -> Result<()> {
 [![regex-badge]][regex] [![cat-text-processing-badge]][cat-text-processing]
 
 Reads a file named `application.log` and only outputs the lines 
-containing “warning” or “error” (case insensitive). A [`regex::RegexSet`] is built
-with [`regex::RegexSetBuilder`] using the builder pattern.
+containing “version X.X.X” or some IP address followed by port 443 (e.g. “192.168.0.1:443”).
 
-```rust
+A [`regex::RegexSet`] is built with [`regex::RegexSetBuilder`].
+Since backslashes are very common in regular expressions, using
+[raw string literals][raw-string-literals] make them more readable.
```

Comment:
no need for link renaming here. you can define the link directly as you would like it displayed at the footer.

`[raw string literals]: https://doc.rust-lang.org/reference/tokens.html#raw-string-literals`